src/actions/appActions.js

```diff
@@ -194,6 +195,9 @@ export const initAppAndRedirectAction = () => {
       const { contacts = [] } = get(storageData, 'localContacts', []);
       dispatch({ type: SET_CONTACTS, payload: contacts });
 
+      const { balances: totalBalances = {} } = get(storageData, 'totalBalances', {});
```

Comment:
Just wanted to keep in the code as other calls, but you know what, maybe this is wrong because I keep copying old pattern every time I add so maybe it's good to add this, thanks